server/routes/finance-id/financeIdController.ts

```diff
@@ -193,9 +178,8 @@ export default class FinanceIdController {
         driversLicenceApplicationMadeAt,
       } = req.body
       const costOfApplication = Number(req.body.costOfApplication)
-      const rpClient = new RPClient(req.user.token, req.sessionID, req.user.username)
       try {
-        await rpClient.post(`/resettlement-passport/prisoner/${prisonerNumber}/idapplication`, {
+        await this.rpService.postIdApplication(`/resettlement-passport/prisoner/${prisonerNumber}/idapplication`, {
```

Comment:
I don't know if this aligns with the current naming convention or not, but wouldn't it be better to call it createIdApplication and updateIdApplication?

Reply:
In general I agree but I think in this case it wouldn't convey new info since a POST should be a create, they are equivalent :)